client/lib/plans/index.js

```diff
@@ -204,3 +215,33 @@ export function plansLink( url, site, intervalType ) {
 
 	return url;
 }
+
+export function applyTestFiltersToPlansList( planName ) {
+	const filteredPlanConstantObj = Object.assign( {}, plansList[ planName ] );
+	let filteredPlanFeaturesConstantObj = getPlanFeaturesConstantObj( plansList[ planName ].getFeatures() );
+
+	if ( ! isWordadsInstantActivationEnabled() ) {
+		filteredPlanFeaturesConstantObj = pickBy( filteredPlanFeaturesConstantObj,
+			( _, key ) => key !== WORDADS_INSTANT
```

Comment:
I used to program in Python and you would use `_` for a throwaway variables. I don't need the `value` here, that's why I used it. But right, will make it `value` to be more readable.